universal-application-tool-0.0.1/app/export/ExporterFactory.java

```diff
@@ -0,0 +1,31 @@
+package export;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import javax.inject.Inject;
+import models.Program;
+import services.program.ExportDefinition;
+
+public class ExporterFactory {
+  @Inject
+  public ExporterFactory() {
```

Comment:
(non-blocking) Guice knows how to use the default constructor so I don't think this is necessary.

Reply:
Yeah, it was just a placeholder, I can add it when I need it, removed.